test/views/datalist/data-list.spec.js

```diff
@@ -179,6 +182,10 @@ describe('Directive:  pfDataList', function () {
       return item.uuid === '2';
     };
 
+    // allow item selection
+    $scope.listConfig.selectItems = true;
+    $scope.listConfig.showSelectBox = false;
+
     $scope.listConfig.checkDisabled = checkDisabled;
```

Comment:
Would it be good to test an error case and that the exception is thrown by specifing:
$scope.listConfig.selectItems = true;
$scope.listConfig.showSelectBox = true;